__stories__/Elevation.stories.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components'
+import StoryGrid from './StoryGrid'
+
+import ElevationUI from '../components/ui/Elevation'
+import { H3 } from '../components/ui/Typography'
+
+const Card = styled(ElevationUI)`
+  display: flex;
+  width: 12rem;
+  align-items: flex-end;
+  padding: 2rem;
+  font-size: 2rem;
+  height: 17.5rem;
+  font-family: ${({ theme }) => theme.fontStacks.chicago};
+`
+
+const Template = ({ Elevation: elevation }) => (
+  <StoryGrid columns={2}>
+    <H3>Fuctional</H3>
```

Comment:
nit: Functional

Reply:
Or you could just skip the functional / static, since this doesn’t appear to be an interactive element, more an applied style.

Reply:
just leaved configurable component